### DISCUSSION TITLE
how is this related to snp in vmware?

### DISCUSSION THREAD

**Discussion by aep** [ccc-certifier-framework/certifier-framework-for-confidential-computing#231] [General]
For a job i'm required to use vsphere instead of kvm which according to the customer already does attestation.
I was looking at this repo hoping to find hints how to extract the attestation from a guest running in vsphere, but havent been able to learn much. Does vpshere use the sev-snp-simulator driver?

**Comment by yelvmw** [comment]
vSphere 8 does not have official support for SEV-SNP yet. It does support SEV-ES without attestation. I would suggest testing things out with the simulator environment we have. You don't need vSphere for that. Any Linux environment where you have permissions to install kernel modules will do.
